src/ui/template/404.html

```diff
@@ -5,9 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Error</title>
-    <link rel="stylesheet" href="/404.css">
+    <link rel="shortcut icon" href="e.ico" type="image/x-icon">
+    <link rel="stylesheet" href="404.css">
```

Comment:
You'll need the initial slash in href="/404.css" because 404 can happen on deep paths. We'll want to pull the 404 from the root path.